src/metrics/mongoose_metrics_probe_cets.erl

```diff
@@ -0,0 +1,83 @@
+-module(mongoose_metrics_probe_cets).
+-behaviour(mongoose_metrics_probe).
+-include("mongoose_logger.hrl").
+
+%% Init the metrics
+-export([start/0]).
+
+%% Probe callbacks
+-export([sample/0, datapoints/0]).
+
+%% GraphQL helper
+-export([format_probe_cets/1]).
+
+start() ->
+    Interval = mongoose_metrics:get_report_interval(),
+    M = [{callback_module, ?MODULE},
+         {sample_interval, Interval}],
+    mongoose_metrics:ensure_metric(global, [cets, system], {probe, M}),
+    ok.
+
+format_probe_cets(Map) ->
+    Acc0 = #{<<"type">> => <<"cets_system">>},
+    F = fun(M, Acc) ->
```

Comment:
Minor, optimization: the whole map is re-created on every KV pair. Same for `all_zeros`.